spk/fuse-exfat/src/installer.sh

```diff
@@ -0,0 +1,64 @@
+#!/bin/sh
+
+# Package
+PACKAGE="exfat"
+DNAME="exfat"
+
+# Others
+INSTALL_DIR="/usr/local/${PACKAGE}"
+
+preinst ()
+{
+    exit 0
+}
+
+postinst ()
+{
+    # Link
+    ln -s ${SYNOPKG_PKGDEST} ${INSTALL_DIR}
+    
+    # Put fuse-exfat & exfat-utils in the PATH
+    mkdir -p /usr/local/bin
+    ln -s ${INSTALL_DIR}/sbin/mount.exfat-fuse /usr/sbin/mount.exfat
```

Comment:
`/usr/sbin` is rewritten during DSM updates, afaik.
In other words, the symlinks are dropped when someone updates DSM (possibly not with all minor versions, but that would depend on what's in the patch).
That's why we're limited to `usr/local`.